tool/state-processor/src/adjust.rs

```diff
@@ -0,0 +1,60 @@
+// darwinia
+use crate::*;
+
+pub trait Adjust {
+	fn adjust(&mut self);
+}
+
+impl Adjust for u32 {
+	fn adjust(&mut self) {
+		*self = self.checked_sub(*NOW.read().unwrap()).unwrap_or_default() / 2;
```

Comment:
Can you explain more about the formula here, why `/2`?

Reply:
https://github.com/darwinia-network/darwinia-2.0/issues/78